docs-2.0/3.ngql-guide/11.edge-statements/3.upsert-edge.md

```diff
@@ -0,0 +1,42 @@
+# UPSERT EDGE
+
+```ngql
+UPSERT EDGE <src_vid> -> <dest_vid> [@rank] OF <edge_type> SET <update_columns> [WHEN <condition>] [YIELD <columns>]
```

Comment:
same question: can we replace "columns" with "properties" or "prop" in this statement?